binder/jupyter_notebook_config.py

```diff
@@ -1,28 +1,53 @@
-lab_command = ' '.join([
-    'jupyter',
-    'lab',
-    '--dev-mode',
-    '--debug',
-    '--extensions-in-dev-mode',
+common = [
     '--no-browser',
+    '--debug',
     '--port={port}',
     '--ServerApp.ip=127.0.0.1',
     '--ServerApp.token=""',
-    '--ServerApp.base_url={base_url}lab-dev',
     # Disable dns rebinding protection here, since our 'Host' header
     # is not going to be localhost when coming from hub.mybinder.org
     '--ServerApp.allow_remote_access=True'
-])
+]
+
+lab_command = ' '.join([
+    'jupyter',
+    'lab',
+    '--dev-mode',
+    '--extensions-in-dev-mode',
+    '--ServerApp.base_url={base_url}lab-dev',
+] + common + ['>jupyterlab-dev.log 2>&1'])
+
+
+lab_splice_command = ' '.join([
+    'jupyter',
+    'lab',
+    'build',
+    '--splice-source',
+    '--minimize=False',
+    '--dev-build=True',
+    '--debug',
+    '>jupyterlab-spliced.log 2>&1'
```

Comment:
Missing comma? (caught by the code scanning check)

```suggestion
    '>jupyterlab-spliced.log 2>&1',
```